shared/i18n/locales/en_US/translation.json

```diff
@@ -654,6 +654,10 @@
   "Allow authorized <em>{{ authenticationMethods }}</em> users to create new accounts without first receiving an invite": "Allow authorized <em>{{ authenticationMethods }}</em> users to create new accounts without first receiving an invite",
   "Default role": "Default role",
   "The default user role for new accounts. Changing this setting does not affect existing user accounts.": "The default user role for new accounts. Changing this setting does not affect existing user accounts.",
+  "Allowed Domains": "Allowed Domains",
+  "The domains which should be allowed to create accounts. This applies to both SSO and Email logins. Changing this setting does not affect existing user accounts.": "The domains which should be allowed to create accounts. This applies to both SSO and Email logins. Changing this setting does not affect existing user accounts.",
+  "Remove Domain": "Remove Domain",
+  "Save Domains": "Save Domains",
```

Comment:
These being auto-generated is _really_ cool! 💯

Reply:
Not yet auto-translated 🤣